pkg/meta/sql.go

```diff
@@ -61,6 +61,12 @@ type edge struct {
 	Type   uint8  `xorm:"notnull"`
 }
 
+type linkParent struct {
+	Id     int64 `xorm:"pk bigserial"`
+	Inode  Ino   `xorm:"notnull"`
```

Comment:
index ?

Reply:
What index? Do you mean count of each parent?

Reply:
We need to find all parents for given `inode`, so `inode` should be a index